docs/setup/clouds/aws.md

```diff
@@ -0,0 +1,59 @@
+# Google Cloud
```

Comment:
Could you add a line like:
```yaml
# mkdocs.yml
- Setup:
  - Clouds:
    - setup/clouds/index.md
    - Gcloud: setup/clouds/gcloud.md
    - Openstack: setup/clouds/openstack.md
    - AWS: setup/clouds/aws.md # this one
...
```
On this [file](https://github.com/scality/runner-manager/blob/main/mkdocs.yml#L51)

Reply:
Done